src/c/device-can.c

```diff
@@ -37,11 +37,6 @@
 		return x.code;                                        \
 	}
 
-#define CAN_INTFC_ADDR "CanInterface"
-#define FILTER_MSG_ID1 "FilterMsgId1"
-#define FILTER_MASK1   "FilterMask1"
-#define TIMEOUT "Timeout"
-
 can_driver *impl;
 int sock_fd;
```

Comment:
`sock_fd` probably should not be a global variable anymore. Maybe move it into `end_dev_params`?

Reply:
@FelixTing , Yes you are right. Thank you. Moved sock_fd to end_dev_params.